<launch>

	<rosparam param="ticks_per_rev">1440</rosparam>
	<rosparam param="ticks_meter">2388</rosparam>
	<rosparam param="maxMetersPerSecond">0.603</rosparam>
	<rosparam param="minMetersPerSecond">0.100</rosparam>
	<rosparam param="motorUpdateHz">1.0</rosparam>
	<rosparam param="rightMotorDirectionSign">1</rosparam>
	<rosparam param="leftMotorDirectionSign">-1</rosparam>
	<rosparam param="leftMotorId">0</rosparam>
	<rosparam param="rightMotorId">1</rosparam>

	<rosparam param="encoder_min">-32767</rosparam>
	<rosparam param="encoder_max">32767</rosparam>
	<rosparam param="encoderUpdateHz">1.0</rosparam>
	<rosparam param="wheel_low_wrap">-32000</rosparam>
	<rosparam param="wheel_high_wrap">32000</rosparam>
	<rosparam param="base_width">0.307</rosparam>

    <node
        pkg="nomad"
        type="diff_baseController.py"
        name="diff_baseController"
        respawn="true"
        />
    
    <node
        pkg="nomad"
        type="diff_wheelEncoders.py"
        name="diff_wheelEncoders"
        respawn="true"
        /> 

    <node
        pkg="joint_state_publisher"
        type="joint_state_publisher"
        name="joint_state_publisher"
        respawn="true"
        />

    <node
        pkg="robot_state_publisher"
        type="state_publisher"
        name="robot_state_publisher"
        respawn="true"
        />

</launch>
